src/test/java/io/reactiverse/pgclient/UnixDomainSocketTest.java

```diff
@@ -95,4 +96,14 @@ public void connectWithVertxInstance(TestContext context) {
       vertx.close();
     }
   }
+
+  @Test
+  public void testIgnoreSslMode(TestContext context) {
```

Comment:
why is SSL mode is ignored with domain sockets ?

Reply:
See the documentation in https://www.postgresql.org/docs/current/libpq-connect.html.
> sslmode is ignored for Unix domain socket communication.

I'm not sure whether Postgres supports SSL over domain socket, maybe we can make domain socket behave the same like TCP connection and let the server handle this?

Reply:
I was actually curious to understand that. I think it's fine this way then as it's not supported at all.